src/kademlia/dht_storage.cpp

```diff
@@ -84,16 +88,16 @@ namespace
 	{
 		// the actual value
 		std::unique_ptr<char[]> value;
+		// size of malloced space pointed to by value
+		int size = 0;
```

Comment:
are you saying that the order matters? didn't know that...changing it to the original version.